docs/operator/deploy.md

```diff
@@ -376,76 +301,48 @@ The JSON-formatted output has the following form:
 }
 ```
 
-The "gid" simply indicates which group the data belongs to. It is present for
-scalar and points on the curve, even though scalars are the same on the three
-groups of bls12-381. The field is present already to be able to accommodate
-different curves later on.
+The "gid" simply indicates which group the data belongs to. It is present for scalar and points on the curve, even though scalars are the same on the three groups of bls12-381. The field is present already to be able to accommodate different curves later on.
 
 ### Chain information
 
-To retrieve the chain information this node participates to, run:
+To retrieve the chain information, this node participates to, run:
 
 ```bash
 drand show chain-info
 ```
 
 ## Updating drand group
 
-Drand allows for "semi-dynamic" group update with a _resharing_ protocol that
-offers the following:
+Drand allows for "semi-dynamic" group update with a _resharing_ protocol that offers the following:
 
-- new nodes can join an existing group and get new shares. Note that, in fact,
-  all nodes get _new_ shares after running the resharing protocol.
-- nodes can leave their current group. It may be necessary for nodes that do
-  not wish to operate drand anymore.
-- nodes can update the threshold associated with their current distributed
-  public key.
+- new nodes can join an existing group and get new shares. Note that, in fact, all nodes get _new_ shares after running the resharing protocol.
+- nodes can leave their current group. It may be necessary for nodes that do not wish to operate drand anymore.
+- nodes can update the threshold associated with their current distributed public key.
```

Comment:
```suggestion
- New nodes can join an existing group and get new shares. Note that, in fact, all nodes get _new_ shares after running the resharing protocol.
- Nodes can leave their current group. It may be necessary for nodes that do not wish to operate drand anymore.
- Nodes can update the threshold associated with their current distributed public key.
```